### DISCUSSION TITLE
为什么没有类似于ctex一样补充定义zhsong等命令来使用\fangsong \heiti等命令

### DISCUSSION THREAD

**Discussion by xkwxdyy** [stone-zeng/fduthesis#238] [Q&A] [answered]
为什么没有像ctex一样补充定义zhsong等，进而更加直接地使用\fangsong \heiti这些命令呢？这样与 ctex也可以统一起来

**Comment by sikouhjw** [comment]
因为在毕业论文中出现这些字体不是很诡异的事情吗？

按照内容与样式分离原则，使用者只需要用 `\emph` 和正文字体就可以了，其它的都不需要使用者去设置字体。

**Comment by stone-zeng** [comment, marked as answer]
模板设计初衷就是想让论文作者回避直接的字体调用（因为这在一般的写作中不是一个常规的需求），特定场合当然可以自行定义。

**Comment by xkwxdyy** [comment]
了解了，我其实是最近在借鉴fduthesis的字体设置写一个CCNUthesis，我本意也不是让用户在正文中使用，只是在改cls的时候用`\ttfamily`这些不太习惯，但参看了`ctex.pdf`和stone-zeng的`fduthesis.pdf`基本知道如何进行对应的字体设置
